TTS/tts/utils/text/phonemizers/espeak_wrapper.py

```diff
@@ -228,8 +228,9 @@ def is_available(cls):
     print(e.name())
     print(e.is_available())
 
-    e = ESpeak(language="en-us", keep_puncs=False)
-    print("`" + e.phonemize("hello how are you today?") + "`")
+    e = ESpeak(language="vi", keep_puncs=False)
+    print("`" + e.phonemize("Tiếng Việt, cũng gọi là tiếng Việt Nam hay Việt ngữ là ngôn ngữ của người Việt.") + "`")
 
+    print("-" * 30)
     e = ESpeak(language="en-us", keep_puncs=True)
-    print("`" + e.phonemize("hello how are you today?") + "`")
+    print("`" + e.phonemize("Tiếng Việt, cũng gọi là tiếng Việt Nam hay Việt ngữ là ngôn ngữ của người Việt.") + "`")
```

Comment:
Maybe use the default English sentence for this second example.

Reply:
Thanks for your reply, I changed the second example to the default English sentence.